lib/ansible/modules/windows/win_chocolatey.ps1

```diff
@@ -176,7 +177,7 @@ Function Choco-Upgrade
         [bool]$ignoredependencies,
         [Parameter(Mandatory=$false, Position=10)]
         [int]$timeout,
-        [Parameter(Mandatory=$false, Position=11)]
+        [Parameter(Mandatory=$false, Position=12)]
```

Comment:
Why did this change, should there be more lines to add `allowmultipleversions` to the 11th position?